reference/configuration/assetic.rst

```diff
@@ -50,7 +50,7 @@ Full Default Configuration
                 some_filter:                 []
             workers:
                 # see https://github.com/symfony/AsseticBundle/pull/119
-                # Cache can also be busted via the framework.templating.assets_version
+                # Cache can also be busted via the framework.assets.version
```

Comment:
Actually, I agree - this should have been in the UPGRADE log (the change of the configuration). Strictly speaking, there are *no* BC breaks on minor version changes, so the UPGRADE log is for exactly stuff like this - big changes where some old way of doing something still works, but is deprecated (check out the Form section of UPGRADE-2.7.md). I've seen a few people complain about the asset stuff being missing from UPGRADE.